pkg/util/validation/validation.go

```diff
@@ -121,6 +121,23 @@ func IsDNS1123Subdomain(value string) []string {
 	return errs
 }
 
+const wildcardDNF1123SubdomainFmt = "\\*\\." + dns1123SubdomainFmt
+
+var wildcardDNS1123SubdomainRegexp = regexp.MustCompile("^\\*\\." + dns1123SubdomainFmt + "$")
+
+// IsWildcardDNS1123Subdomain tests for a string that conforms to the definition of a
+// wildcard subdomain in DNS (RFC 1034) and (RFC 1123).
```

Comment:
does the rfc specific the exact regex we're using, or is it an approximation of what we translate the rfc to mean?